skytemple_files/data/waza_p/_model.py

```diff
@@ -179,12 +179,12 @@ def __init__(self, data: bytes):
         self.is_snatchable = bool(read_u8(data, 0x11))
         # 0x12	1	uint8	Unk#15	Boolean, whether the move is disabled by the "muzzled" status.
         self.uses_mouth = bool(read_u8(data, 0x12))
-        # 0x13	1	uint8	Unk#16	Unknown.
-        self.unk13 = read_u8(data, 0x13)
+        # 0x13	1	uint8	Unk#16	If true, the AI won't try to use the move on frozen targets.
+        self.ai_frozen_check = read_u8(data, 0x13)
         # 0x14	1	uint8	Unk#17	Boolean, whether the move can be used while taunted.
         self.ignores_taunted = bool(read_u8(data, 0x14))
-        # 0x15	1	uint8	Unk#18	Unknown. Possible bitfield.
-        self.unk15 = read_u8(data, 0x15)
+        # 0x15	1	uint8	Unk#18	Determines the string that is displayed for the range of the move in-game
```

Comment:
You could clarify that the value is offset by a certain amount here, but I guess it's not too important.